packages/docusaurus-utils/src/index.ts

```diff
@@ -173,19 +173,19 @@ export function normalizeUrl(rawUrls: string[]): string {
     urls[0] = first + urls[0];
   }
 
-  // There must be two or three slashes in the file protocol, two slashes in anything else.
-  if (urls[0].match(/^file:\/\/\//)) {
-    urls[0] = urls[0].replace(/^([^/:]+):\/*/, '$1:///');
-  } else {
-    urls[0] = urls[0].replace(/^([^/:]+):\/*/, '$1://');
-  }
+  /**
+   * There must be two or three slashes in the file protocol,
+   * two slashes in anything else.
+   */
+  let replaceWith = urls[0].match(/^file:\/\/\//) ? '$1:///' : '$1://';
+  urls[0] = urls[0].replace(/^([^/:]+):\/*/, replaceWith);
 
   // eslint-disable-next-line
   for (let i = 0; i < urls.length; i++) {
     let component = urls[i];
 
     if (typeof component !== 'string') {
-      throw new TypeError(`Url must be a string. Received ${component}`);
+      throw new TypeError(`Url must be a string. Received typeof ${component}`);
```

Comment:
IMO this makes more sense as per the context :thinking:

Reply:
Shouldn't the typeof be within the `${ ... }`?

Reply:
oops